portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,6 +86,22 @@ private String convertToJsonUsingGson(List messages) {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
+    //TODO: Move into its own function.
```

Comment:
This is mostly my opinion: This TODO seems pretty quick to do (would likely take O(minutes)). So if the thing you're leaving a TODO for is a worthwhile change, and it won't take long, just do whatever it is instead of leaving a TODO. Oftentimes TODOs get added and never addressed.

I personally like to save TODOs for the following things:
- Something that would be nice to have (e.g. a performance improvement), but it would take a while to do and isn't worth doing right now.
- Marking areas that I will need to change in future to make sure I don't forget to follow up (e.g. something I can't do yet because I'm waiting for X to be implemented first).
- If I'm having troubles creating small pull requests, sometimes I will leave out a part of the implementation and mark it with a TODO, then include that implementation in a different pull request. That said, I try to avoid this when possible since there is usually a better way to make the pull request smaller.

Reply:
Thanks for the advice! I will take it into account next time. As for this TODO, I took out because I put that portion of code someplace else and don't think separation is needed.